pkg/cmd/pipelineresource/list.go

```diff
@@ -81,13 +83,17 @@ func listCommand(p cli.Params) *cobra.Command {
 				return fmt.Errorf("failed to list pipelineresources. Invalid resource type %s", opts.Type)
 			}
 
-			pres, err := list(cs.Resource, p.Namespace(), opts.Type)
+			namespace := p.Namespace()
+			if opts.AllNamespaces {
+				namespace = ""
+			}
+			pres, err := list(cs.Resource, namespace, opts.Type)
 			stream := &cli.Stream{
 				Out: cmd.OutOrStdout(),
 				Err: cmd.OutOrStderr(),
 			}
 			if err != nil {
-				fmt.Fprintf(os.Stderr, "Failed to list pipelineresources from %s namespace \n", p.Namespace())
+				fmt.Fprintf(os.Stderr, "Failed to list pipelineresources from %s namespace \n", namespace)
```

Comment:
This will show up with double space in case of all-namespace

Reply:
agreed I will update this accordingly